internal/api/generate.go

```diff
@@ -85,7 +85,7 @@ func (g Generate) Action(ctx *cli.Context) error {
 		dir = path.Join(modulePaths, cfg.Generate.DependencyEntryPoint.Path)
 	}
 
-	err = generator.Generate(ctx.Context, ".", dir)
+	err = generator.Generate(ctx.Context, ".", dir, cfg.Generate)
```

Comment:
Extend `Generator` struct instead of passing `cfg.Generate` as function' argument?

Reply:
Moved generate config to generator.